### PR TITLE
feat: Add a possibility to simulate push notifications via simctl

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
https://betterprogramming.pub/how-to-send-push-notifications-to-the-ios-simulator-2988092ba931

### CODE REVIEW

**Review by wswebcreation — APPROVED** [review]
Wow,

This is really great and thanks for doing it so quickly!

🚀

File: lib/commands/notifications.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * @since Xcode SDK 11.4
+ * @param {PushNotification} opts - The object that describes Apple push notification content.
+ * It must contain a top-level `bundleId` key with a string value matching
+ * the target application‘s bundle identifier and "payload.aps" key with valid Apple Push Notification values.
+ * Check the output of `xcrun simctl help push` command for more details.
+ */
+extensions.mobilePushNotification = async function mobilePushNotification (opts = {}) {
+  if (!this.isSimulator()) {
+    throw new Error('This extension only works on Simulator');
```
Comment (KazuCocoa) [inline lib/commands/notifications.js:28]: 👍

File: README.md [hunk 2]
```diff
@@ -623,6 +623,20 @@ Name | Type | Required | Description | Example
...
+Only application remote push notifications are supported. VoIP, Complication, File Provider,
+and other types are not supported. Check the output of `xcrun simctl help push`
+command for more details.
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+bundleId | string | yes | The bundle identifier of the target application | com.apple.Preferences
+payload | map | yes | Valid Apple Push Notification values. Read the `Create the JSON Payload` topic of the [official Apple documentation](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification?language=objc) for more details on the payload creation. | `{"aps": {"alert": "This is a simulated notification!", "badge": 3, "sound": "default"} }`
```
Comment (KazuCocoa) [inline README.md:638]: 👍

**Review by KazuCocoa — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (gromanas) [reply README.md:638]: If I could made a comment, I have already try to use `{"aps": {"alert": "This is a simulated notification!", "badge": 3, "sound": "default"} }` and i got an error about `Simulator Target Bundle`. I have change it to `{"Simulator Target Bundle": "your.apps.bundleId", "aps": {"alert": { "title": "This is a simulated notification!"}, "badge": 3, "sound": "default"} }` and it worked.

**Pull request merged**